React/finger-exercises/react/ecmascript/01 - spread/index.js

```diff
@@ -0,0 +1,28 @@
+import isArray from './utils';
+
+export function min(...data) {
+  if (data[0] === undefined) {
+    return (data[0]);
+  } else if (isArray(...data)) {
+    return (Math.min(...data[0]));
+  } else if (isArray(data)) {
+    return (Math.min(...data));
+  }
+}
```

Comment:
as a general comment for this function... too many `if/ else if` try to find a way to evaluate all of these conditions without so many `if else` statements

Reply:
an 'else if' is removed and the if condition is modified